dask/dataframe/tests/test_io.py

```diff
@@ -712,3 +753,10 @@ def test_hdf_globbing():
         tm.assert_frame_equal(res.compute(), pd.concat([df] *  3))
     finally:
         shutil.rmtree(tdir)
+
+
+if __name__ == "__main__":
+    test_read_csv_with_datetime_index_partitions_one()
+    test_read_csv_with_datetime_index_partitions_n()
+    test_from_pandas_with_datetime_index()
+    test_usecols()
```

Comment:
We should avoid these.  Instead I run tests by the following

    mrocklin@notebook:~/workspace/dask$ py.test dask/dataframe/tests/test_io.py